src/application/extension-popup/popup-home-view/app.tsx

```diff
@@ -16,37 +16,44 @@ import { SectionTile } from './components';
 export const App = () => {
   const navigate = useNavigate();
   return (
-    <div className="flex max-w-[570px] grow flex-col items-center bg-gray-100 px-1 pt-6">
-      <header className="mb-6 text-center">
-        <h1 className="mb-2 text-xl font-bold text-black">
+    <div className="flex max-w-[470px] grow flex-col items-center bg-gray-100 px-1 py-4">
+      <header className="mb-4 text-center">
+        <h1 className="mb-1 text-xl font-bold text-black">
           GM, it&apos;s your web3 copilot speaking
         </h1>
         <p className="text-sm text-gray-600">
-          Explore the features below for your best experience
+          Explore your favorite features below
         </p>
       </header>
 
-      <div className="grid w-full grid-cols-2 gap-4 px-4">
+      <div className="grid w-full grid-cols-2 gap-3 px-3">
         <SectionTile
           href="https://x.com/IDriss_xyz/status/1729258820441170160"
+          target="_blank"
+          rel="noopener noreferrer"
           icons={[{ iconSrc: IDRISS_ICON, iconAlt: 'Tipping Icon' }]}
           title="Tipping Badges"
           subtitle="Send crypto on X and Farcaster"
         />
 
         <SectionTile
           href="https://x.com/IDriss_xyz/status/1761049574691819798"
+          target="_blank"
+          rel="noopener noreferrer"
           icons={[
-            { iconSrc: AGORA_LOGO, iconAlt: 'Agora Logo' },
-            { iconSrc: SNAPSHOT_LOGO, iconAlt: 'Snapshot Logo' },
-            { iconSrc: TALLY_LOGO, iconAlt: 'Tally Logo' },
+            { iconSrc: AGORA_LOGO, iconAlt: 'Agora Logo', className: 'w-6 h-6' },
+            { iconSrc: SNAPSHOT_LOGO, iconAlt: 'Snapshot Logo', className: 'w-6 h-6' },
+            { iconSrc: TALLY_LOGO, iconAlt: 'Tally Logo', className: 'w-6 h-6' },
           ]}
           title="Governance Widget"
-          subtitle="Get reminders about active DAO votes"
+          subtitle="Get alerts about active DAO votes"
+          iconContainerClassName="mt-2"
         />
 
         <SectionTile
           href="https://x.com/IDriss_xyz/status/1780582103396266085"
+          target="_blank"
```

Comment:
We can get rid of the target and rel here. The ones added inside for the <a /> tag should be enough

Reply:
ok, I added it as I think opening in new tab wasnt working without it